docs/collecting-data/collecting-from-own-applications/mobile-trackers/migration-guides/migration-guide-to-new-ecommerce/index.md

```diff
@@ -0,0 +1,194 @@
+
+tracker.track(event)
+```
+
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+Old API:
+
+```java
+EcommerceTransactionItem product = new EcommerceTransactionItem(
+    "sku1", // sku
+    100.0, // price
+    1 // quantity
+  ).name("itemName")       
+  .category("category")       
+  .currency("GBP")
+  .orderId("id-123")
+
+List<Product> products = new ArrayList<>();
+products.add(product);
+
+EcommerceTransaction event = EcommerceTransaction(
+      "id-123", // orderId
+      350.0, // totalValue
+      products // items
+  ).currency("GBP")
+  .taxValue(10.0)
+  .shipping(15.0)
+  .city("London")
+  .state("N/A")
+  .country("UK")
+  .affiliation("affiliation")
+
+tracker.track(event)
+```
+New API:
+
+```java
+ProductEntity product = new ProductEntity(
+  "sku1", // id
+  100.0, // price
+  1, // quantity
+  "itemName", // name
+  "category", // category
+  "GBP", // currency
+);
+List<Product> products = new ArrayList<>();
+products.add(product);
+
+TransactionEntity transaction = new TransactionEntity(
+  "id-123", // id
+  350.0, // revenue
+  products, // products
+  "GBP", // currency
+  10.0, // tax
+  15.0 // shipping
+);
+
+TransactionEvent event = new TransactionEvent(transaction, products);
+
+tracker.track(event)
+```
+  </TabItem>
+</Tabs>
+
+These code snippets show only the direct API replacement. Check out the [docs page](docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/ecommerce-tracking/index.md) or the [Ecommerce Accelerator](https://snowplow.io/data-product-accelerators/ecommerce-analytics-dpa/) for information about expanding your tracking with the other new event types.
+
+Note that address details and affiliation are not captured in `TransactionEvent`. To track these so that they are attached to the `TransactionEvent`, we suggest creating [custom schemas](docs/collecting-data/collecting-from-own-applications/mobile-trackers/custom-tracking-using-schemas/index.md) and adding the data as an entity. Alternatively for addresses, you could use the `CheckoutStepEvent` which has address and postcode as optional parameters.
+
+## How the tracked events are different
+
+Using the old event types, an event is generated for each `EcommerceTransaction` event as well as each `EcommerceTransactionItem` attached. Therefore, tracking an `EcommerceTransaction` event with 3 items would result in 4 events being tracked. These events are not linked at all, unless the same `orderId` is provided to all the objects, adding potential challenges in data modeling.
+
+In the tracked data, the events have `event_name` `transaction` or `transaction_item`. The data is sent within the legacy event properties `tr_x` and `ti_x`, e.g. `tr_id` for the `EcommerceTransaction` `orderId` property. All of these properties are listed [here](docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/index.md). Each one maps to a column in the warehouse table.
```

Comment:
This bit confused me slightly - is this talking about the previous Ecommerce data? (I assume so based on the next few lines, but it wasn't clear until after this line).

Perhaps the `How the tracked events are different` section could do with a couple of subheaders like "Previous Ecommerce events/data" and "New ecommerce events" or something?